website/docs/d/privatelink_endpoint_service_data_federation_online_archives.html.markdown

```diff
@@ -22,7 +22,7 @@ resource "mongodbatlas_project" "atlas-project" {
 
 resource "mongodbatlas_privatelink_endpoint_service_data_federation_online_archive" "test" {
   project_id = mongodbatlas_project.atlas-project.id
-  endpoint_id = "<PRIVATE-ENDPOINT-SERVICE-ID>"
```

Comment:
Also @EspenAlbert can you update examples in 2 Data Sources file to include `customer_endpoint_dns_name` and `endpoint_id` parameters

Reply:
Changes made in 7c55d85 ✅